data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -276,6 +316,72 @@ def get_sandbox_dataset_id(self):
         """
         return self._sandbox_dataset_id
 
+    @property
+    def affected_tables(self):
+        """
+        Method to get tables that will be modified by the cleaning rule.
+        If getting the tables_affected dynamically logic needs to be
+        implemented in this method.
+        """
+        return self._affected_tables
+
+    @affected_tables.setter
+    def affected_tables(self, affected_tables):
+        """
+        Set the affected_tables for this class instance
+        """
+        if affected_tables:
+            self._affected_tables = affected_tables
+        else:
```

Comment:
This is defaulting to a list when None.  We should note it in the docstring and enforce it in the value check.  If a value is passed to the setter that is not None and is not a list, an error should be raised.